toe/package.json

```diff
@@ -1,15 +1,16 @@
 {
-  "name": "tic-tac-toe",
-  "version": "0.0.1",
-  "description": "TIC-TAC-TOE for Urbit",
-  "main": "npm run build",
+  "name": "create-landscape-app",
```

Comment:
Yeah, I'm not sure now which version of the package is getting, or should go in here, tbh

Reply:
The `package.json` describes the code that's in the folder, so this should definitely get a custom name instead of the placeholder `create-landscape-app`.